### PR TITLE
Compatibility fixes for PHPC and HHVM

### PR DESCRIPTION
**Pull request by jmikola**

### CODE REVIEW

File: src/Operation/ListCollections.php [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 namespace MongoDB\Operation;
 
+require 'src/functions.php';
+
```
Comment (jmikola) [inline src/Operation/ListCollections.php]: Composer should handle autoloading and requiring `src/functions.php`, so this shouldn't be necessary. Can you try regenerating your autoloaders again with `composer dump-autoload` or `composer update`?

File: tests/Collection/BulkWriteFunctionalTest.php [hunk 2]
```diff
@@ -35,7 +35,7 @@ public function testInserts()
             array('_id' => $insertedIds[1], 'x' => 22),
         );
 
-        $this->assertEquals($expected, $this->collection->find()->toArray());
+        $this->assertSameDocuments($expected, $this->collection->find());
```
Comment (jmikola) [inline tests/Collection/BulkWriteFunctionalTest.php]: Good idea. I thought about creating such an assertion.

File: src/Operation/ListCollections.php (hunk 1 above)
Comment (derickr) [inline src/Operation/ListCollections.php]: Oh, you're probably right here. Let me see.

**Comment by jmikola** [comment]
@derickr: Please let me know if this works for you.

**Pull request merged**